src/main/java/org/jabref/gui/autocompleter/StringStringConverter.java

```diff
@@ -0,0 +1,18 @@
+package org.jabref.gui.autocompleter;
+
+import javafx.util.StringConverter;
+
+public class StringStringConverter extends StringConverter<String> {
+
+    public StringStringConverter() { }
+
+    @Override
+    public String toString(String string) {
+        return string;
+    }
+
+    @Override
+    public String fromString(String string) {
+        return string;
+    }
```

Comment:
This seems to be a bit silly, doesn't it? Maybe there is a generic-converter for it in javax.util.stringconverter?